fix(versions): update Activiti/activiti-cloud-dependencies versions

jx-activiti-cloud opened the pull request:
[UpdateBot](https://github.com/jenkins-x/updatebot) pushed make dependency: `ACTIVITI_CLOUD_RB_VERSION` to: `7.1.258`

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.258

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_RB_VERSION 7.1.258

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies 7.1.149 org.activiti.cloud.examples:example-runtime-bundle 7.1.258

pow-devops2020 committed:
fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.149

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_QUERY_VERSION 7.1.214

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.214

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.query:activiti-cloud-query-dependencies 7.1.132 org.activiti.cloud.examples:activiti-cloud-query 7.1.214

pow-devops2020 committed:
fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.132

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_RB_VERSION 7.1.259

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.259

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies 7.1.150 org.activiti.cloud.examples:example-runtime-bundle 7.1.259

pow-devops2020 committed:
fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.150

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.227

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_NOTIFICATIONS_VERSION to 7.1.227

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.227 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.132

pow-devops2020 committed:
fix(version): update org.activiti.cloud.examples:activiti-cloud-notifications-graphql to 7.1.227

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make MODELING_DEPENDENCIES_VERSION 7.1.232 ACTIVITI_CLOUD_MODELING 7.1.440

pow-devops2020 committed:
fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.232

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_RB_VERSION 7.1.260

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.260

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies 7.1.151 org.activiti.cloud.examples:example-runtime-bundle 7.1.260

pow-devops2020 committed:
fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.151

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make MODELING_DEPENDENCIES_VERSION 7.1.233 ACTIVITI_CLOUD_MODELING 7.1.441

pow-devops2020 committed:
fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.233

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_QUERY_VERSION 7.1.215

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.215

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.query:activiti-cloud-query-dependencies 7.1.133 org.activiti.cloud.examples:activiti-cloud-query 7.1.215

pow-devops2020 committed:
fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.228

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.228 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.229

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.229 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.230

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.230

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.230

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.229

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.232

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.232 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.233

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_NOTIFICATIONS_VERSION to 7.1.233

igdianov committed:
fix(tests):  corrected version property for activiti-cloud-notifications-graphql

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.233 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.233 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.133

pow-devops2020 committed:
fix(version): update org.activiti.cloud.examples:activiti-cloud-notifications-graphql to 7.1.233

almerico committed:
Update Makefile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Makefile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Makefile

almerico committed:
Update Makefile

almerico committed:
Update Makefile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Makefile

almerico committed:
Update Jenkinsfile

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_QUERY_VERSION 7.1.216

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.216

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.query:activiti-cloud-query-dependencies 7.1.134 org.activiti.cloud.examples:activiti-cloud-query 7.1.216

pow-devops2020 committed:
fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.134

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_RB_VERSION 7.1.261

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.261

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies 7.1.152 org.activiti.cloud.examples:example-runtime-bundle 7.1.261

pow-devops2020 committed:
fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.152

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make MODELING_DEPENDENCIES_VERSION 7.1.234 ACTIVITI_CLOUD_MODELING 7.1.442

pow-devops2020 committed:
fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.234

almerico committed:
Update Makefile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Jenkinsfile

almerico committed:
Update Jenkinsfile

almerico committed:
fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.258

fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.149

fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.214

fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.132

fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.259

fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.150

fix(version): update ACTIVITI_CLOUD_NOTIFICATIONS_VERSION to 7.1.227

fix(version): update org.activiti.cloud.examples:activiti-cloud-notifications-graphql to 7.1.227

fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.232

fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.260

fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.151

fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.233

fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.215

fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.133

fix(version): update ACTIVITI_CLOUD_NOTIFICATIONS_VERSION to 7.1.233

fix(tests):  corrected version property for activiti-cloud-notifications-graphql


fix(version): update org.activiti.cloud.examples:activiti-cloud-notifications-graphql to 7.1.233

Update Makefile
Update Jenkinsfile
Update Makefile
Update Jenkinsfile
Update Jenkinsfile
Update Jenkinsfile
Update Makefile
Update Makefile
Update Makefile
Update Jenkinsfile
Update Makefile
Update Jenkinsfile
fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.216

fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.134

fix(version): update ACTIVITI_CLOUD_RB_VERSION to 7.1.261

fix(version): update org.activiti.cloud.rb:activiti-cloud-runtime-bundle-dependencies to 7.1.152

fix(version): update MODELING_DEPENDENCIES_VERSION to 7.1.234

Update Makefile
Update Jenkinsfile
Update Jenkinsfile
Update Jenkinsfile

almerico committed:
Merge remote-tracking branch 'origin/updatebot-5a411d26-1a68-4259-b6ef-f2e32192cfd2' into updatebot-5a411d26-1a68-4259-b6ef-f2e32192cfd2

almerico committed:
Update Makefile

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_QUERY_VERSION 7.1.217

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_QUERY_VERSION to 7.1.217

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.query:activiti-cloud-query-dependencies 7.1.135 org.activiti.cloud.examples:activiti-cloud-query 7.1.217

pow-devops2020 committed:
fix(version): update org.activiti.cloud.query:activiti-cloud-query-dependencies to 7.1.135

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind make ACTIVITI_CLOUD_NOTIFICATIONS_VERSION 7.1.234

pow-devops2020 committed:
fix(version): update ACTIVITI_CLOUD_NOTIFICATIONS_VERSION to 7.1.234

ghost commented:
[UpdateBot](https://github.com/jenkins-x/updatebot) commands:

    updatebot push-version --kind maven org.activiti.cloud.examples:activiti-cloud-notifications-graphql 7.1.234 org.activiti.cloud.notifications.graphql:activiti-cloud-notifications-graphql-dependencies 7.1.134

pow-devops2020 committed:
fix(version): update org.activiti.cloud.examples:activiti-cloud-notifications-graphql to 7.1.234

almerico committed:
Update Jenkinsfile
